pages/edit/collection/[id].js

```diff
@@ -0,0 +1,32 @@
+import CollectionForm from "@/components/CollectionForm";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const StyledHeader = styled.h1`
+  display: flex;
+  justify-content: center;
+`;
+
+export default function EditCollection({ collections, onSubmitCollection }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const selectedCollection = collections.find(
+    (collection) => collection.id === id
+  );
+
```

Comment:
please add a check if a collection was found. if not return an error message. You then don't have to provide fallback data to the prevValues prop and can instead pass down the collection directly. `prevValues={selectedCollection}`.